server/src/tenant/query-builder/factories/composite-field-alias.factory.ts

```diff
@@ -98,7 +98,7 @@ export class CompositeFieldAliasFactory {
 
     // Otherwise it means it's a relation destination is of kind ONE
     return `
-      ${fieldKey}: ${targetTableName} {
+      ${fieldKey} {
```

Comment:
@Weiko I don't think it's a good solution, because user can still choose a custom name for the relation